test/xpu_api/numerics/complex.number/complex.ops/complex_divide_complex.pass.cpp

```diff
@@ -71,7 +71,7 @@ void test_edges()
                 case zero:
 #if !_PSTL_TEST_COMPLEX_DIV_COMPLEX_BROKEN_GLIBCXX
                     assert(classify(r) == inf);
-#endif
+#endif // _PSTL_TEST_COMPLEX_DIV_COMPLEX_BROKEN_GLIBCXX
```

Comment:
Is this necessary?  the `#if` is 2 lines away.  This is also somewhat unrelated to this PR. (same for the others in this file)

If we add this, there are some other ones like line 82 and 103 we should also add for consistency (but I'd prefer just removing them).